types/staticmaps/index.d.ts

```diff
@@ -107,6 +108,26 @@ declare namespace StaticMaps {
         | 'start'
         | 'middle'
         | 'end';
+
+    interface AddCircleOptions {
+        coord: [number, number];
+        radius: number;
+        /**
+         * Stroke color of the circle
+         * @default '#000000BB'
+         */
+        color?: string;
```

Comment:
```suggestion
        color?: string | undefined;
```
be consistent, unless there is a reason to skip undefined here